### PR TITLE
Fix camera lookAt in orbit controls

### PR DESCRIPTION
**Pull request by devfans**
I might be wrong, but looks it's not using the correct way to lookAt back to the target. The orbit should only change the state in local coordinate, so could be bug if not on purpose

### CODE REVIEW

File: examples/jsm/controls/OrbitControls.js [hunk 1]
```diff
@@ -211,8 +214,12 @@ var OrbitControls = function ( object, domElement ) {
 			offset.applyQuaternion( quatInverse );
 
 			position.copy( scope.target ).add( offset );
-
-			scope.object.lookAt( scope.target );
```
Comment (Mugen87) [inline examples/jsm/controls/OrbitControls.js:215]: Sorry but I'm afraid your change is incorrect. I suggest you study how [Object3D.lookAt()](https://github.com/mrdoob/three.js/blob/a14f405825b938da3a1a072e2f9084441a671689/src/core/Object3D.js#L266-L306) is implemented. You should then see that your new code is not necessary.

Comment (devfans) [reply examples/jsm/controls/OrbitControls.js:215]: yeah, I was checking the lookAt implementation, which is calculating with the worldMatrix indeed. I am implementing my own controls and working out after apply this change. So guessing this should be a bug.

Comment (devfans) [reply examples/jsm/controls/OrbitControls.js:215]: The issue without the change is if you orbit around the target while you are moving the character, you wold see the target is losing focus from the camera. Or it's a desired behavior if I was misunderstanding the OrbitControl?

Comment (WestLangley) [reply examples/jsm/controls/OrbitControls.js:215]: @devfans See the discussion in https://github.com/mrdoob/three.js/pull/16374.

**Pull request closed without merging**